packages/rum/src/viewCollection.ts

```diff
@@ -55,6 +56,20 @@ export function startViewCollection(location: Location, lifeCycle: LifeCycle, se
     currentView.triggerUpdate()
     currentView.end()
   })
+
+  // Session keep alive
+  const keepAliveInterval = setInterval(
+    monitor(() => {
+      currentView.triggerUpdate()
+    }),
+    SESSION_KEEP_ALIVE_INTERVAL
+  )
+
+  return {
+    stop() {
+      clearInterval(keepAliveInterval)
```

Comment:
Maybe we should end the current view here

Reply:
cf first commit

Reply:
Mmh the first commit is related to not updating the view on session renew.. I don't see how it's related? We could end the view without updating it.

Reply:
my bad, I was thinking about `view.end`

the view collection stop is only called in unit test, what would be the goal of stopping it here? how would it work for session renewed, path change?

Reply:
You are right, this would be only for tests. Also yes, we should probably stop listening for session renewed events and path change.

Unsubscribing from all listeners and ending the current view would have the same goal as clearing the keepalive interval: make sure no event is sent after stoping view collection. This is optional (even stoping the keepalive interval is not needed), because we never use the same Lifecycle object for two tests. But it seems weird to me to stop one thing and not the other.

Reply:
oh, did you mean "Maybe we should **also** end the current view here"?

Reply:
Yes :)